src/app.controller.spec.js

```diff
@@ -0,0 +1,34 @@
+import { Test } from "@nestjs/testing";
+import { AppController } from "./app.controller";
+import { AppService } from "./app.service";
+
+describe("AppController", () => {
+  let app;
+
+  beforeAll(async () => {
+    app = await Test.createTestingModule({
+      controllers: [AppController],
+      providers: [AppService],
+    }).compile();
+  });
+
+  describe("getReadingTime test01", () => {
+    it("should return 90,000ms", () => {
+      const appService = app.get(AppService);
+      const userWPM = 186;
+      const articleBody =
+        "연합뉴스 육군 제12사단에서 규정에 어긋난 군기 훈련(얼차려)을 받다 숨진 박모 훈련병의 수료식이 19일 열린 가운데 ‘상관이 시킨다고 무조건 듣지 말라’고 당부하는 입대 장병 부모가 늘고 있다. 아들을 고(故) 박 훈련병과 함께 입대했다고 밝힌 아버지 A씨는 최근 JTBC 유튜브 ‘뉴스들어가혁’과 인터뷰에서 “아들을 (수료식에서) 만나면 ‘그냥 시키는 것만 하고 나서지 말라’ ‘절대 건강하라’는 말을 해주고 싶다”면서 아들을 비롯해 군 복무하는 모든 장병에게 ‘너무 힘들면 영창에 갈 것을 각오하고라도 상관의 명령을 거부하라’는 취지로 발언했다. A씨는 “건강은 너희가 지켜야지 아무도 챙겨주지 않는다(는 말을 해주고 싶다)”면서 “입대할 때는 (군에 보낸 아들이) 대한민국의 군인이라고 그렇게 부모들에게 자랑하더니 무슨 사고만 터지면 ‘당신 아들’이라며 외면을 하니 누가 자식을 믿고 군에 보내겠느냐”고 목소리를 높였다. 고 박 훈련병 관련 소식을 전한 국민일보 뉴스에도 “5만 군 가족이 지켜보고 있다. 하루하루 애국심이 바닥으로 떨어진다” “멀쩡한 청년이 국방의 의무를 이행하러 갔다가 지휘관에 의해 고문당해 죽은 것이 이 사건의 본질이다” “군에 무늬만 군인인 국방 조무사로 가득하다. 뒤집어엎어야 한다” 등 군 당국을 비판하는 댓글이 잔뜩 달렸다. 한편 이날 고 박 훈련병의 어머니 B씨는 군인권센터를 통해 아들이 군기 훈련을 받은 상황과 쓰러진 뒤 군의 조치에 대해 문제를 제기했다. 그는 “군이 아들에게 씌운 프레임은 ‘떠들다 (규칙을 어겨) 얼차려를 받은 것’이었는데 아들이 동기들과 실제로 나눈 말은 ‘조교를 하면 아침에 일찍 일어나야겠다’와 같은 것들이었다”면서 “내 아들과 잔악한 선착순 달리기를 시키고 언제 끝날지 모르는 구보를 뛰게 하다 아들을 쓰러뜨린 중대장 중 누가 규칙을 더 많이 어겼느냐”고 지적했다. 강원경찰청 훈련병 사망 사건 수사 전담팀은 사건 발생 26일 만인 지난 18일 해당 중대장과 부중대장에게 직권남용가혹행위와 업무상과실치사 혐의를 적용해 검찰에 구속 영장을 신청했다. 김진욱 기자 reality@kmib.co.kr 경제부 김진욱 기자 reality@kmib.co.kr 응원하기 금융권 소식을 전합니다. GoodNews paper ⓒ 국민일보(www.kmib.co.kr), 무단전재, 수집, 재배포 및 AI학습 이용 금지 ";
+
+      expect(appService.getReadingTime(articleBody, userWPM)).toBe(90000);
+    });
+
+    it("should return 90,000ms", () => {
```

Comment:
```suggestion
    it("should return 60,000ms", () => {
```

60,000ms이 더 적합해보입니다.